src/test/java/com/aws/greengrass/deployment/DeploymentServiceTest.java

```diff
@@ -407,6 +408,8 @@ void GIVEN_deployment_job_WHEN_deployment_process_succeeds_THEN_correctly_map_co
         allGroupTopics.children.put(new CaseInsensitiveString(expectedGroupName), deploymentGroupTopics);
 
         when(config.lookupTopics(GROUP_TO_LAST_DEPLOYMENT_TOPICS)).thenReturn(groupToLastDeploymentTopics);
+        lenient().when(config.lookupTopics(eq(GROUP_TO_LAST_DEPLOYMENT_TOPICS), anyString())).thenReturn(
```

Comment:
Why lenient here?

Reply:
This test runs for Jobs, Shadow and Local deployment types. This lookup is only done for iot job deployments, in other two that srub is not used, so I had to put lenient.